emp-wrapper/src/main/java/org/ballerinalang/sf/ListenerUtil.java

```diff
@@ -132,7 +170,67 @@ private static long getReplayFrom(ObjectValue service) {
     }
 
     private static BError sfdcError(String errorMessage) {
-        return ErrorCreator.createDistinctError(Constants.SFDC_ERROR, Constants.PACKAGE_ID_SFDC, StringUtils.fromString(errorMessage));
- 
+        return ErrorCreator.createDistinctError(Constants.SFDC_ERROR, PACKAGE_ID_SFDC,
+                StringUtils.fromString(errorMessage));
     }
+
+    /**
+     * Convert Map to BMap.
+     *
+     * @param map Input Map used to convert to BMap.
+     * @return Converted BMap object.
+     */
+    public static BMap<BString, Object> toBMap(Map map) {
+        BMap<BString, Object> returnMap = ValueCreator.createMapValue();
+        if (map != null) {
+            for (Object aKey : map.keySet().toArray()) {
+                returnMap.put(StringUtils.fromString(aKey.toString()),
+                        StringUtils.fromString(map.get(aKey).toString()));
+            }
+        }
+        return returnMap;
+    }
+
+//    /**
+//     * Convert Map to Ballerina record tpe
+//     *
+//     * @param event Input Map used to convert to BMap.
+//     * @return Converted BMap object.
+//     */
+//    private static BMap<BString, Object> getEventDataRecord(Map<String, Object> event)  {
```

Comment:
Why do we have commented out code? Shall we remove?

Reply:
The code sample is for mapping the JSON response to a ballerina record type in java interoperability. It was not successful because changing the parameters in the payload. Will remove that.